Documentation/op-guide/monitoring.md

```diff
@@ -0,0 +1,75 @@
+# Monitoring etcd
+
+etcd exports all its metrics through client port under `/metrics` path.
+
+Example output
+
+```
+$ curl -L http://localhost:2379/metrics
+
+# HELP etcd_debugging_mvcc_keys_total Total number of keys.
+# TYPE etcd_debugging_mvcc_keys_total gauge
+etcd_debugging_mvcc_keys_total 0
+# HELP etcd_debugging_mvcc_pending_events_total Total number of pending events to be sent.
+# TYPE etcd_debugging_mvcc_pending_events_total gauge
+etcd_debugging_mvcc_pending_events_total 0
+...
+```
+
+
+## Prometheus
+
+Easiest way to consume those metrics is [Prometheus][prometheus] monitoring system.
```

Comment:
Running a Prometheus monitoring service is the easiest way to ingest and record etcd's metrics.